### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by zamotany** [zamotany/logkitty#1]
# Roadmap

- [ ] support for multiple Android devices
- [ ] support for iOS devices

**Comment by friederbluemle** [comment]
Does this Roadmap need to be updated? `0.6.0` is out already. ;)

**Comment by zamotany** [comment]
@friederbluemle yeah it was out of date. I've updated it. Apart from those tasks I'm not sure what else could be done in `logkitty`. If you have any ideas, feel free to send it my way.
